src/main/java/org/candlepin/subscriptions/tally/facts/NormalizedFacts.java

```diff
@@ -80,6 +81,14 @@ public void setOwner(String owner) {
         this.owner = owner;
     }
 
+    public String getHypervisorUuid() {
+        return hypervisorUuid;
+    }
+
+    public void setHypervisorUuid(String hypervisorUuid) {
+        this.hypervisorUuid = hypervisorUuid;
+    }
+
```

Comment:
Please add a comment stating what this property represents.It may not be clear to someone reading the code.

Reply:
Will do.